dev/merge_spark_pr.py

```diff
@@ -72,6 +72,13 @@
 BRANCH_PREFIX = "PR_TOOL"
 
 
+def print_error(msg):
+    print("\033[91m%s\033[0m" % msg)
+
+
+def bold_input(prompt) -> str:
+    return input("\033[1m%s\033[0m" % prompt)
+
```

Comment:
nit. We need two empty new lines, don't we?

Reply:
fixed